### DISCUSSION TITLE
Copilot VS extension is missing code

### DISCUSSION THREAD

**Discussion by Symbai** [community/community#70797] [Copilot Conversations]
### Select Topic Area

Bug

### Body

The VS extension is missing code from suggestions. VS Code doesnt have this problem. Im basically following the code I saw on YT https://youtu.be/XceIKXeIrD0?t=152

Here you can see that the suggestions ends with "else if (value > node"
![image](https://github.com/community/community/assets/61151836/38e07723-57cc-4f16-b010-f9be576f4230)

If I now click on "Accept solutions", the code is cut off here as well:
![image](https://github.com/community/community/assets/61151836/26c43f96-09c7-440b-9d7a-3f7269884589)

I'm fine with the preview window has a text limit (although I wish it would not). But when accept the solution it should be the FULL solution / code.

Copilot Version: 1.126.0.0
VS Community: 2022 - 17.8.0 Preview 4

**Comment by ayim** [comment]
@Symbai just messaged a developer on a team to investigate this behaviour.

**Reply by ayim** [reply]
@Symbai update: it looks like we have a suggestion length limit. Next steps here are a design discussion & experiment to see if we would consider changing this behaviour

**Reply by Symbai** [reply]
Thank you for the update. If the text limit is important for any reason, I think a good idea is to auto complete the "unfinished" code like it already does but then immediately suggest the rest of the code. So the extension has to check if the added code is incomplete and triggers a new suggestion by itself (or fetches the rest of the code).

However the VS code extension has added the full code like you can see in the Youtube video above. But the code preview in VS code extension is also scrollable and the preview in VS extension is not. Is that why there is a text limit in VS extension? Anyway I'm sure you'll find a good solution.
